.travis/dispatch.sh

```diff
@@ -53,6 +53,6 @@ else
     # Only run tests against the new configuration
     # TODO: don't hard-code this
     if [[ "${SALT_NODE_ID}" == "servo-master1" ]]; then
-        ./test.py sls.buildbot.master sls.homu sls.nginx
+        ./test.py sls.buildbot.master sls.common.timezone sls.homu sls.nginx
```

Comment:
Two things:
- this can be just `sls.common`, all of the test files in that dir will get run automatically.
- we want to run the common tests on all nodes, not just `servo-master1`, so that should happen outside the conditional.